apps/kg/dataloader/sampler.py

```diff
@@ -221,9 +223,9 @@ def __init__(self, dataset, args):
                     pickle.dump(g, graph_file)
         self.g = g
 
-        self.num_train = len(dataset.train)
-        self.num_valid = len(dataset.valid)
-        self.num_test = len(dataset.test)
+        self.num_train = len(dataset.train[0])
```

Comment:
what is the different between dataset.train and dataset.train[0]. Why it is changed here?

Reply:
originally, `dataset.train` stores a list of tuples. Each of the tuples represents an edge. Now `dataset.train` stores a tuple of numpy ndarray.